<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QDialog" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>457</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>457</width>
    <height>515</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="calcTable">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>445</width>
     <height>380</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>445</width>
     <height>380</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>445</width>
     <height>380</height>
    </size>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculation">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Ставка</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SummCred">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximum">
    <double>999999999.990000009536743</double>
   </property>
   <property name="value">
    <double>2000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="TermCred">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
   <property name="value">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BidCred">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="Annuitet">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>115</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>115</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Different">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>115</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>115</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Дифференц.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Сброс</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Переплата:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ResCalcPrec">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>480</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Всего:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ResCalcCred">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
